ztp/ran-crd/site-config-crd.yaml

```diff
@@ -393,3 +393,10 @@ spec:
                                       name:
                                         description: nic name used in the yaml, which relates 1:1 to the mac address. Name in REST API logicalNICName
                                         type: string
+                            inspect:
```

Comment:
Can we use something more descriptive than "inspect". For someone who is learning about deploying sites they will ask the question "what am I inspecting on the node?". Maybe bmcInspect, baremetalInspect, ???

Reply:
It refers to the Ironic 'inspection' process. I will change it to  'ironicInspect'.

Reply:
Done